staff/federico-lacabaratz/precourse/calculator-html/index.js

```diff
@@ -0,0 +1,138 @@
+
+// En primer lugar declaro las variables de operaciones
+var opera1;
+var opera2;
+
+// variables para teclas de la calculadora
+var uno = document.getElementById('one');
+var result = document.getElementById('displayNum');
+var allClear = document.getElementById('ac');
+var clear = document.getElementById('cl');
+var suma = document.getElementById('sum');
+var resta = document.getElementById('res');
+var multiplica = document.getElementById('mul');
+var divide = document.getElementById('div');
+var igual = document.getElementById('equal');
+var punto = document.getElementById('dot');
+var dos = document.getElementById('two');
+var tres = document.getElementById('three');
+var cuatro = document.getElementById('four');
+var cinco = document.getElementById('five');
+var seis = document.getElementById('six');
+var siete = document.getElementById('seven');
+var ocho = document.getElementById('eight');
+var nueve = document.getElementById('nine');
+var cero = document.getElementById('zero');
+
+// funciones relacionadas con los eventos de los botones de la calculadora
+
+uno.onclick = function(event) {
+    result.value = result.value += "1";
+}
+
+dos.onclick = function(event) {
+    result.value = result.value += "2";
+}
+
+tres.onclick = function(event) {
+    result.value = result.value += "3";
+}
+
+cuatro.onclick = function(event) {
+    result.value = result.value += "4";
+}
+
+cinco.onclick = function(event) {
+    result.value = result.value += "5";
+}
+
+seis.onclick = function(event) {
+    result.value = result.value += "6";
+}
+
+siete.onclick = function(event) {
+    result.value = result.value += "7";
+}
+
+ocho.onclick = function(event) {
+    result.value = result.value += "8";
+};
+
+nueve.onclick = function(event) {
+    result.value = result.value += "9";
+};
+
+cero.onclick = function(event) {
+    result.value = result.value += "0";
```

Comment:
Good Evening Federico, nice job with the calculator but, why don't use only one function to assign the value of the click in the var result like this? 
`document.getElementById("digit").addEventListener("click",addvalue)
function addvalue() {         
        result.placeholder += this.value
}`
i hope you help

Reply:
Hi Kike! I read your english is very good mate!!.

I´ve done basically the same thing as you proposed but instead of writing a whole statement for each number, I called the element by using the "document.getElement...." at the beginning of the JS and store it inside a variable (this I did for all numbers). Then I simply called the variables and asign the "onclick" method and made it work. Cheers!